webservices/AuthService.c

```diff
@@ -992,7 +999,7 @@ gsi_bool wsLoginCertIsValid(const GSLoginCertificate * cert)
                                   if(lenoutSoFar + _tcslen(a) > maxlen) \
                                       return gsi_false; \
 								  strcpy(bufout+lenoutSoFar, a); \
-								  lenoutSoFar += _tcslen(a) + 1; }
+								  lenoutSoFar += (unsigned int)_tcslen(a) + 1; }
```

Comment:
please use gsi_u32 for all the casts in lenoutSoFar

Reply:
https://github.com/GameProgressive/UniSpySDK/pull/27/commits/fafe1ceeb163b4ba77da8c57e3dec9a3f2ad6e2d